content/blog/2020/02/2020-02-10-t-mobile-case-study.adoc

```diff
@@ -0,0 +1,4 @@
+---
+layout: redirect
+redirect_url: /node/tags/pipeline/
```

Comment:
I would redirect to a 404 page

Reply:
Thanks.  Fixed in 6f18d28f43efa32ae4c412781cef5c46212e0753